llvm/test/CodeGen/NVPTX/f16-abs.ll

```diff
@@ -0,0 +1,99 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+; ## Some FP16 support but not for abs
+; RUN: llc < %s -mtriple=nvptx64-nvidia-cuda -mcpu=sm_53  \
```

Comment:
Few nits on the tests -- I would explicitly set PTX version high enough on all test cases to make sure that native `abs` instruction is disabled by the GPU type or by explicit attributes, and not unintentionally so by the too-low default PTX version.

Reply:
Okay, I've updated the cases to include each of the following:
- no f16 abs due to insufficient hardware
- no f16 abs due to insufficient ptx version
- no f16 abs due to explicit disable
- f16 abs supported

Reply:
Apologies, I did not, I've just pushed 0c7af950b7dbefcb3ee545ddd74670ffcb6d7869